lib/action_controller/serialization/context.rb

```diff
@@ -0,0 +1,12 @@
+module ActionController
```

Comment:
It's not in action_controller, it's namespaced within actioncontroller::serialization which is fine by me